README.md

```diff
@@ -792,6 +824,7 @@ Optionally, you can include the following as a URL query parameter:
 | Field     | Type    | Description |
 |-----------|---------|-------------|
 | validated | Boolean | If `true`, the server waits to respond until the account transaction has been successfully validated by the network. A validated transaction has `state` field of the response set to `"validated"`. |
+| submit | Boolean | If `false`, the server will not submit the transaction to the network and instead return the following additional fields in the response: `tx_json`, `tx_blob`, and `hash`. In this mode, `secret` is optional. If `secret` is not provided, `tx_blob` and `hash` will not be in the response; these can be obtained by sending the `tx_json` value to the [`sign`](#sign-transaction) endpoint. After persisting the `hash`, the `tx_blob` value can be sent to the [`submit`](#submit-transaction) endpoint to submit the transaction to the network. |
```

Comment:
This is a lot of information for a table cell. I'd move the parts about the response to the section that describes the response.